aspnet/web-forms/overview/moving-to-aspnet-20/data-bound-controls.md

```diff
@@ -662,19 +662,21 @@ This exercise assumes that you have SQL Server running locally and that the Nort
 8. Select **Northwind** in the connection dropdown and click Next.
 9. Select **Products** from the **Name** dropdown and check the **ProductID**, **ProductName**, **UnitPrice**, and **UnitsInStock** checkboxes as shown below. 
 
-![](data-bound-controls/_static/image3.jpg)
+![A screenshot of the configure data source - products screen.](data-bound-controls/_static/image3.jpg)
```

Comment:
A screenshot of the configure data source products screen. 
Removed the dash for screen readers.